apps/mdn/mdn-aws/k8s/Makefile

```diff
@@ -203,6 +206,9 @@ export NEW_RELIC_MONITOR_MODE ?= 1
 export NEW_RELIC_LOG ?= stderr
 export NEW_RELIC_BROWSER_MONITOR_ENABLE ?= false
 export NEW_RELIC_LABELS ?= Project:MDN;Cluster:${K8S_CLUSTER_SHORT_NAME};Namespace:${K8S_NAMESPACE}
+export NEW_RELIC_HIGH_SECURITY ?= true
```

Comment:
Thanks, this is a requirement for Mozilla web projects, to avoid sending user-identifiable data through New Relic.